iOS job-description

marijnschilling opened the pull request:
I copied what we have now, with some minor changes to make it at least not wrong :)

And i added the "what you'll be doing" part, bluntly copied from colin's adroid description 🎉

Let's make it good quick so we can hire the best!

marijnschilling committed:
iOS job-description

3lvis commented on HIRING.md:
```diff
@@ -48,3 +48,26 @@ We’re obviously trying to assess the candidate but remember they are assessing
+# Job description for an iOS position
+
+### A bit about you
+- 01 Experience with building top level applications for Apple iOS/OS X, and started peeping into Watch OS and tvOS.
```
Let's keep it simple.
`01 Experience with building top level applications for iOS.`

marijnschilling replied:
I agree, I really don't care if you've started peeping into tv or watch :shrug:

elland replied:
No one cares about tv or watch. 🗡

3lvis commented on HIRING.md:
```diff
@@ -48,3 +48,26 @@ We’re obviously trying to assess the candidate but remember they are assessing
+### A bit about you
+- 01 Experience with building top level applications for Apple iOS/OS X, and started peeping into Watch OS and tvOS.
+- 02 Extensive experience with team and client work.
+- 03 A broad understanding of Swift, Core Data and Web-services.
```
This could be:

```
03 A broad understanding of Swift, Core Data and RESTful APIs.
```

elland committed:
Update HIRING.md (#82)

* Update HIRING.md

Changed the language to be more inclusive, and the requirements to fit more with what we do. :)

* Update HIRING.md

* Update HIRING.md

elland commented on HIRING.md:
```diff
@@ -47,4 +47,26 @@ We’re obviously trying to assess the candidate but remember they are assessing
+### A bit about you
+- 01 Familiarity with building applications for iOS.
+- 02 Teamplayer.
+- 03 Familiarity or interest in Swift, usability and Web services.
```
how about `such as REST and Websockets` here?

marijnschilling committed:
Update HIRING.md (#83)

* Update HIRING.md

* Update HIRING.md
